src/neuroglancer/widget/voxel_size_selection_widget.ts

```diff
@@ -0,0 +1,113 @@
+import {TrackableMIPLevelConstraints} from 'neuroglancer/trackable_mip_level_constraints';
+import {RefCounted} from 'neuroglancer/util/disposable';
+import {removeFromParent} from 'neuroglancer/util/dom';
+import {vec3} from 'neuroglancer/util/geom';
+import './voxel_size_selection_widget.css';
+
+export class VoxelSizeSelectionWidget extends RefCounted {
```

Comment:
For special use cases where the currently active MIP level is important, it would be useful to show the active MIP level somewhere here in the widget.